packages/Webkul/Shop/src/Http/Controllers/Customer/RegistrationController.php

```diff
@@ -57,7 +57,7 @@ public function store(RegistrationRequest $registrationRequest)
             'is_verified'               => ! core()->getConfigData('customer.settings.email.verification'),
             'customer_group_id'         => $this->customerGroupRepository->findOneWhere(['code' => 'general'])->id,
             'token'                     => md5(uniqid(rand(), true)),
-            'subscribed_to_news_letter' => request()->input('is_subscribed') ?? 0,
+            'subscribed_to_news_letter' => request()->input('is_subscribed') ? '1' : '0',
```

Comment:
The following will work too, don't use unnecessary ternary operator

(boolean) request()->input('is_subscribed')